layers/exporters/to_excel.py

```diff
@@ -0,0 +1,120 @@
+from openpyxl.comments import Comment
+from openpyxl.styles import PatternFill, Font
+
+try:
+    from core import Layer
+    from exporters import ExcelTemplates
+except ModuleNotFoundError:
+    from ..core import Layer
+    from ..exporters import ExcelTemplates
+
+class NoLayer(Exception):
+    pass
+
+class ToExcel:
+    def __init__(self, domain='enterprise', server=False, local=None):
+        """
+            Sets up exporting system, builds underlying matrix
+
+            :param server: Whether or not to use live cti-taxii data to construct the matrix
+            :param local: Optional path to local stix data
+
+        """
+        self.raw_handle = ExcelTemplates(domain=domain, server=server, local=local)
+
+    def to_xlsx(self, layer, filepath="layer.xlsx"):
+        """
+            Exports a layer file to the excel format as the file specified
+
+            :param layer: A layer to initialize the instance with
+            :param filepath: The location to write the excel file to
+        """
```

Comment:
Should probably throw an error if you input a layer with a domain different than that specified in the constructor. Keep in mind that layer domains are formatted weirdly (e.g `mitre-enterprise` instead of `enterprise-attack`). We're going to fix this inconsistency later this year but it's probably months away at this point.

Reply:
Got it.